src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs

```diff
@@ -2,27 +2,31 @@
   (:require-macros [status-im.utils.views :as views])
   (:require [re-frame.core :as re-frame]
             [status-im.ui.components.react :as react]
-            [status-im.ui.screens.profile.user.views :as profile]
             [status-im.utils.build :as build]
-            [status-im.utils.utils :as utils]
             [status-im.ui.components.colors :as colors]
             [status-im.i18n :as i18n]
             [status-im.ui.components.icons.vector-icons :as vector-icons]
-            [taoensso.timbre :as log]
             [clojure.string :as string]
             [status-im.ui.screens.offline-messaging-settings.views :as offline-messaging.views]
             [status-im.ui.components.qr-code-viewer.views :as qr-code-viewer]
-            [status-im.ui.screens.desktop.main.tabs.profile.styles :as styles]
-            [status-im.ui.screens.profile.user.views :as profile]))
+            [status-im.ui.screens.desktop.main.tabs.profile.styles :as styles]))
 
-(defn profile-badge [{:keys [name photo-path]}]
+(defn profile-badge [{:keys [name photo-path]} editing?]
   [react/view styles/profile-badge
    [react/image {:source {:uri photo-path}
                  :style  styles/profile-photo}]
-   [react/text {:style           styles/profile-user-name
-                :font           :medium
-                :number-of-lines 1}
-    name]])
+   (if editing?
+     [react/text-input {:auto-focus    true
+                        :default-value name
```

Comment:
It's not this line's problem frankly, but neither `defaultValue` nor `value` properties are support in RN-desktop yet.

Reply:
Can be temporarily replaced by `:placeholder` I guess

Reply:
https://github.com/status-im/react-native-desktop/issues/332

Reply:
Have no idea why TextInput is not clickable though...

Reply:
@vkjr @MaxRis maybe you might have idea what's up there

Reply:
When I look at screenshot provided by @chadyj above, first thought is that text-input has no proper size information.

As for missed `TextInput` properties, `value` exists for sure. And I believe it should work fine, I was checking it few weeks ago.

Reply:
@vkjr I agree and retract my statement about `value` field. Sorry, missed it (

As for missing size information - I was experimenting with adding `flex: ` to the TextInput component for the profile name, and in both cases (with and without `flex`) user name was visible just fine in my case. Not sure how to reproduce Chad's problem.

Reply:
@vkjr I've re-checked things and updated issue title/description for 
https://github.com/status-im/react-native-desktop/issues/332.

Indeed the component becomes clickable if width and height are set (but it should not be necessary). However, what is strange is that for instance in desktop add-new screen input components [link](https://github.com/status-im/status-react/blob/bd3f9570b30462a60989b9a77c8f4d4514936191/src/status_im/ui/screens/desktop/main/add_new/views.cljs#L34) no width and height need to be set for them to be visible and working properly.